src/agent/io/sourcemapper.ts

```diff
@@ -38,7 +38,23 @@ export interface MapInfoInput {
 export interface MapInfoOutput {
   file: string;
   line: number;
-  column?: number;
+  column: number;
+}
+
+/** Interfaces pulled from source-map-support. */
+interface Position {
```

Comment:
Is there a reason we can't use these directly from source-map-support?  Are they not exported from it?